ogre2/src/Ogre2ThermalCamera.cc

```diff
@@ -231,6 +244,26 @@ void Ogre2ThermalCameraMaterialSwitcher::preRenderTargetUpdate(
           }
         }
       }
+      // see if a heat signature exists
+      else if (auto heatSignature = std::get_if<std::string>(&tempAny))
+      {
+        // TODO(adlarkin) set the texture in the material to heatSignature
+        // before applying the material? Right now, the texture is hardcoded,
+        // so heatSignature is unused
```

Comment:
could be that the texture is not in ogre's resource path. If that's the case, this should help:
https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering4/ogre2/src/Ogre2Material.cc#L439